src/Service/SaveIllustration.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Service;
+
+class SaveIllustration{
+
+    public function save($illustration, $photoDir) {
```

Comment:
photodir can be an attribute that you configure, and then no need to pass it as a parameter each time